oceanprefilter/oceanprefilter.go

```diff
@@ -300,18 +305,32 @@ func (pf *prefilter) ClassificationsFromCamera(
 func (pf *prefilter) Classifications(ctx context.Context, img image.Image,
 	n int, extra map[string]interface{},
 ) (classification.Classifications, error) {
-	select {
-	case <-ctx.Done():
-		return nil, errors.Wrap(ctx.Err(), "module might be configuring")
-	case <-pf.cancelContext.Done():
-		return nil, errors.Wrap(pf.cancelContext.Err(), "lost connection with background camera stream loop")
-	default:
+	if pf.camName == "" {
```

Comment:
I don't think you have to wrap this in an if statement -- even if a background loop is running with a camera, that shouldn't prevent someone from also using any image they want